src/python/pants/backend/jvm/register.py

```diff
@@ -157,6 +158,7 @@ def register_goals():
   task(name='ivy', action=IvyResolve).install('resolve', first=True)
   task(name='ivy-imports', action=IvyImports).install('imports')
   task(name='unpack-jars', action=UnpackJars).install()
+  task(name='ivy', action=IvyUpdates).install('updates')
```

Comment:
I feel like there could be a better name than `updates`. Bundler from the ruby ecosystem uses [outdated](http://bundler.io/v1.14/man/bundle-outdated.1.html) for their command. I think that could work.

I could imagine the use case for this would be something like:

```
./pants outdated 3rdparty::
[updates]
  [python]
    Updates available! There are N outdated artifacts
    abcd:1.2.3 -> abcd:1.2.4
    ...
  [ivy]
     Updates available! There are N outdated artifacts
     xyz:1.2.3 -> xyz:1.2.4
     ...
  SUCCESS
```